core/src/main/java/org/apache/calcite/rex/RexExecutorImpl.java

```diff
@@ -56,11 +56,19 @@
 public class RexExecutorImpl implements RexExecutor {
 
   private final DataContext dataContext;
+  private final SqlConformance sqlConformance;
 
   public RexExecutorImpl(DataContext dataContext) {
     this.dataContext = dataContext;
+    this.sqlConformance = SqlConformanceEnum.DEFAULT;
```

Comment:
I don't think conformance belongs here. Conformance governs validation behavior, not runtime behavior.